spacy_annotator/__init__.py

```diff
@@ -135,6 +138,50 @@ def __add_annotation(self, df, col_text, current_index, annotations):
                 continue
         entities = {"entities": spans}
         df.at[current_index, "annotations"] = (df[col_text][current_index], entities)
+    
+    def to_spacy(self, df, file_path: str):
```

Comment:
I would move this method _after_ the `annotate` method to keep the flow of the class from load data, to annotate to export in spacy format.